data_fixes/subject_remediation/replace_subjects.rb

```diff
@@ -0,0 +1,46 @@
+require 'archivesspace/client'
+require 'active_support/all'
+require_relative '../../helper_methods.rb'
+
+@client = aspace_staging_login
+
+puts Time.now
+
+csv = CSV.parse(File.read("topical_United_States.csv"), :headers=>true, :return_headers=>true)
```

Comment:
Where does this file come from? Is it supposed to come from the parse_United_states.rb file? I don't see that file writing a CSV

Reply:
No, I created it locally.